bots/lending-liquidator/README.md

```diff
@@ -0,0 +1,27 @@
+# lending-liquidator
+
+Liquidation bot for the lending protocol on the Kintsugi and Interlay parachains. The bot only liquidates loans that can be repaid directly with its balance (swapping on the DEX is not yet supported).
```

Comment:
Suggestion: Add a little bit more info on the motivation behind liquidating lending positions. eg. Profitability, or margins that can be realized.

In isolation, it isn't clear if this is a bot that "patrols" the ecosystem to liquidate loans, or if there is another motivation behind maybe running your own one.